client/src/components/Account/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,88 @@
+import { Box, List, ListItem, ListItemText, Typography } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+import { useLocation } from 'react-router';
+
+import { useStyles } from './useStyles';
+
+export default function SideBar(): JSX.Element {
+  const classes = useStyles();
+  const { pathname } = useLocation();
+  return (
+    <Box maxWidth={250} className={classes.sideBarContainer}>
+      <List className={classes.sideBar}>
+        <ListItem>
+          <Link
+            to="/user/editProfile"
```

Comment:
Convention for url paths would be to hyphenate rather than use camelcase, ditto throughout

Reply:
I was on the wall whether to use `kebab-case` or `camelCase`.  In an earlier review, it was suggested that I go with camelCase but I have now changed it to kebab-case. Thanks!